coforma/swift-tech-challenge
# Issue/Create a Header Component

# Title
Issue/Create a Header Component

# Summary

This creates a basic header component using the new SASS and USWDS integration. 

Fixes #16 

https://github.com/coforma/swift-tech-challenge/assets/19439679/146095d5-6b4f-4048-a7b8-73fe9d8cdf5c

## Type of change


# How To Test

1. run `yarn dev`, then navigate to `localhost:3000` to verify the app is running successfully.

# Checklist:
